tests/test_run_state.py

```
@@ -0,0 +1,41 @@
+        run_id = f"run-{i}"
+        run_dir = tmp_path / run_id
+        run_dir.mkdir()
+        state = {
+            "run_id": run_id,
+            "flow_id": "test-flow",
+            "status": "completed",
+            "created_at": "2023-01-01T00:00:00Z"
+        }
+        (run_dir / "run_state.json").write_text(json.dumps(state))
+
+    async def _test():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_test`

Add return type annotation: `None`

(ANN202)